### PR TITLE
Octane dev latest 5.4.1 beta - changes

### PR DESCRIPTION
**Pull request by m-seldin**
- [Rebranding HPE - Microfocus](https://issues.jenkins-ci.org/browse/JENKINS-49989)

- [UFT  integration - rename test in the repository](https://issues.jenkins-ci.org/browse/JENKINS-52030)

- [user story #622117 : UFT integration - add to discovery job configuration - add timestamps to the console output](https://issues.jenkins-ci.org/browse/JENKINS-52031)

- user story #597069:when branch is deleted then the corresponding pipeline in octane should be deleted. add item listener and sending event to octane when branch is deleted

- user story #622057 when github-branch-source plugin is used to manage multibranch then Octane should auto manage pipeline (same as pipeline multibranch plugin)

https://issues.jenkins-ci.org/browse/JENKINS-51881
https://issues.jenkins-ci.org/browse/JENKINS-51882

- Fix log dispatching and improve logs (defect #634035 JIRA-51860 )
Increase sdk version

### CODE REVIEW

**Comment by m-seldin** [comment]
@YafimK 
please merge

File: src/main/java/com/hpe/application/automation/tools/octane/executor/TestExecutionJobCreatorService.java [hunk 1]
```diff
@@ -272,7 +274,7 @@ private static FreeStyleProject getDiscoveryJob(DiscoveryInfo discoveryInfo) {
             if (proj == null) {
 
                 proj = Jenkins.getInstance().createProject(FreeStyleProject.class, discoveryJobName);
-                proj.setDescription(String.format("This job was created by the HPE Application Automation Tools plugin for discovery of %s tests. It is associated with ALM Octane testing tool connection #%s.",
+                proj.setDescription(String.format("This job was created by the Microfocus Application Automation Tools plugin for discovery of %s tests. It is associated with ALM Octane testing tool connection #%s.",
```
Comment (grount) [inline src/main/java/com/hpe/application/automation/tools/octane/executor/TestExecutionJobCreatorService.java]: Micro Focus not Microfocus
- [ ] please ask docs to verify :)

File: src/main/java/com/hpe/application/automation/tools/octane/executor/scmmanager/ScmPluginFactory.java [hunk 2]
```diff
@@ -76,6 +76,20 @@ public static ScmPluginHandler getScmHandlerByScmPluginName(String pluginName) {
         return getScmHandler(scmType);
     }
 
+    public static ScmPluginHandler getScmHandlerByChangePathClass(String  changePathClass) {
+        SCMType scmType = null;
+
+        if ("hudson.plugins.git.GitChangeSet$Path".equals(changePathClass)) {
+            scmType = scmType.GIT;
+        } /*else if ("hudson.scm.SubversionSCM".equals(pluginName)) {
```
Comment (grount) [inline src/main/java/com/hpe/application/automation/tools/octane/executor/scmmanager/ScmPluginFactory.java]: Remove redundant commented code

Comment (m-seldin) [reply src/main/java/com/hpe/application/automation/tools/octane/executor/scmmanager/ScmPluginFactory.java]: done

**Comment by m-seldin** [comment]
@grount done

**Comment by grount** [comment]
Approved for test build

**Pull request merged**